src/components/Form/SetAclPermissionForm.jsx

```diff
@@ -59,37 +53,21 @@ const SetAclPermissionForm = () => {
     try {
       await setDocAclPermission(session, docType, permissions, podUrl, webIdToSetPermsTo);
 
-      runNotification(
-        `${permissions.read ? 'Give' : 'Revoke'} permission to ${
-          permissionState.podUrlToSetPermissionsTo
-        } for ${docType}.`,
-        5,
-        state,
-        dispatch
-      );
       addNotification(
         'success',
         `${permissions.read ? 'Gave' : 'Revoked'} permission to ${
           permissionState.podUrlToSetPermissionsTo
         } for ${docType}.`
       );
     } catch (error) {
-      runNotification('Failed to set permissions. Reason: File not found.', 5, state, dispatch);
       addNotification('error', 'Failed to set permissions. Reason: File not found.');
     } finally {
-      setTimeout(() => {
-        clearInputFields();
-      }, 3000);
+      setTimeout(() => {}, 3000);
```

Comment:
don't think this does anything or is needed.

Reply:
Think we could keep this for a new `setProcessing(false)` for the button?